docs/Translation.md

```diff
@@ -180,6 +180,7 @@ You can find translation packages for the following languages:
 - German (`de`): [greenbananaCH/ra-language-german](https://github.com/greenbananaCH/ra-language-german) (tree translation: [straurob/ra-tree-language-german](https://github.com/straurob/ra-tree-language-german))
 - Hebrew (`he`): [ak-il/ra-language-hebrew](https://github.com/ak-il/ra-language-hebrew)
 - Hungarian (`hu`): [phelion/ra-language-hungarian](https://github.com/phelion/ra-language-hungarian)
+- Hindi (`hi`): [harshit-budhraja/ra-language-hindi](https://github.com/harshit-budhraja/ra-language-hindi)
```

Comment:
The list is ordered alphabetically. Would you mind inserting your package one line earlier?

Reply:
Hi @fzaninotto, sure! My bad. I did notice that the list is lexicographically sorted. Might have made an error. I'll just quickly update the PR.

Reply:
Done!